instantsearch/src/main/java/com/algolia/instantsearch/helpers/InstantSearch.java

```diff
@@ -245,20 +323,6 @@ public void registerFilters(List<AlgoliaFilter> filters) {
         }
     }
 
-    /**
-     * Gets the {@link android.support.annotation.IdRes IdRes} of the item layout registered by a Hits widget.
-     * This method should only be used by the library's internals.
-     *
-     * @return the registered item layout id, if any.
-     * @throws IllegalStateException if no item layout was registered.
-     */
-    public static int getItemLayoutId() {
```

Comment:
make sure you're not breaking BC here

Reply:
Fair concern! However, as there was no reason to use it from a user's perspective, and as I warned in the docs `This method should only be used by the library's internals`, I'm confident this is a harmless change :)